src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector128.cs

```diff
@@ -2384,7 +2384,7 @@ public static Vector128<sbyte> Shuffle(Vector128<sbyte> vector, Vector128<sbyte>
         [CompExactlyDependsOn(typeof(AdvSimd))]
         [CompExactlyDependsOn(typeof(AdvSimd.Arm64))]
         [CompExactlyDependsOn(typeof(PackedSimd))]
-        internal static Vector128<byte> ShuffleUnsafe(Vector128<byte> vector, Vector128<byte> indices)
+        public static Vector128<byte> ShuffleUnsafe(Vector128<byte> vector, Vector128<byte> indices)
```

Comment:
Is there anywhere else this should be consumed in dotnet/runtime as part of making it public?

Reply:
I believe I updated all such places when adding the internal helper as part of #80963

Reply:
Ok.  I know you updated places in corelib.  I was wondering if there were places outside of corelib you couldn't update.  Sounds like no.

Reply:
There are two places I saw where we still use the platform-specific instructions directly ([BitArray](https://github.com/dotnet/runtime/blob/1380a0b327a24a924ef3f63f144558c742da0a3c/src/libraries/System.Collections/src/System/Collections/BitArray.cs#L844-L852) and [OptimizedInboxTextEncoder](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Text.Encodings.Web/src/System/Text/Encodings/Web/OptimizedInboxTextEncoder.Ssse3.cs)), but both of those are in larger platform-specific blocks of logic, so there was no reason to replace them.